src/plugins/embeddable/public/lib/containers/embeddable_child_panel.tsx

```diff
@@ -56,7 +56,7 @@ interface State {
 export class EmbeddableChildPanel extends React.Component<EmbeddableChildPanelProps, State> {
   [panel: string]: any;
   public mounted: boolean;
-  public embeddable!: IEmbeddable | ErrorEmbeddable;
+  public embeddable!: Embeddable | ErrorEmbeddable;
```

Comment:
@streamich This change  is necessary to provide access to `embeddable.actionStorage.count` but breaks code that expects an `IEmbeddable`.

Its not entirely clear to me how we choose when we pass `IEmbeddables` vs `Embeddables`.  I'm tempted to change usage to `Embeddable` as  necessary but there might be implications that I don't yet understand so I'd like your input.